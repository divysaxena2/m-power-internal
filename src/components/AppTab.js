import React, { Component } from 'react';
import getTheme from '../../native-base-theme/components'
import material from '../../native-base-theme/variables/material';
import { StyleSheet, View } from 'react-native';
import { StyleProvider, Tab, Tabs, Text } from 'native-base';



type Props = {};
export default class AppTab extends Component<Props> {

    render() {
        return (
            <StyleProvider style={getTheme(material)}>
                <Tabs tabBarUnderlineStyle={{ height: 7, borderRadius: 3.5 }}>
                    <Tab heading="Credit Card/Debit Card">
                    </Tab>
                    <Tab heading="Bank Account">
                    </Tab>
                </Tabs>
            </StyleProvider>)
    }
}
const styles = StyleSheet.create({


});