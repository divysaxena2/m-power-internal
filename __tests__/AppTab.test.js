import 'react-native';
import React from 'react';
import AppTab from '../src/components/AppTab';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <AppTab />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});