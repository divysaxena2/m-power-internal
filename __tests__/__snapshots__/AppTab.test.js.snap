// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "borderBottomWidth": 0.5,
          "borderColor": "#004987",
          "elevation": 3,
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-around",
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.2,
          "shadowRadius": 1.2,
        },
        Object {
          "backgroundColor": "#FEFFFF",
        },
        Object {},
      ]
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flex": 1,
          "opacity": 1,
        }
      }
    >
      <View
        active={true}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#FEFFFF",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#003A64",
                "fontFamily": "Roboto",
                "fontSize": 16,
                "fontWeight": "600",
                "marginLeft": 7,
                "marginRight": 7,
              },
              Object {
                "fontSize": 13,
              },
              undefined,
            ]
          }
          uppercase={false}
        >
          Credit Card/Debit Card
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flex": 1,
          "opacity": 1,
        }
      }
    >
      <View
        active={false}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#FEFFFF",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#000000",
                "fontFamily": "Roboto",
                "fontSize": 16,
                "marginLeft": 7,
                "marginRight": 7,
              },
              Object {
                "fontSize": 13,
              },
              undefined,
            ]
          }
          uppercase={false}
        >
          Bank Account
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#004987",
          "bottom": 0,
          "height": 4,
          "left": 0,
          "position": "absolute",
          "width": 375,
        }
      }
    />
  </View>
</View>
`;
